bool value
---
# Success if empty
string error
